spec/games_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'spec_helper'
+require './lib/games'
+
+RSpec.describe Games do
+    before(:each) do
+        @game_objects = []
+
+        #iterate over the csv file to create an array of game objects
+        CSV.foreach('./spec/fixtures/fixture_games.csv', headers: true, header_converters: :symbol) do |row|
+            @game_objects << Games.new(
+              row[:game_id],
+              row[:season],
+              row[:away_team_id],
+              row[:home_team_id],
+              row[:away_goals].to_i,
+              row[:home_goals].to_i,
+            )
+        end
+    end
+
+    describe '#initialize' do
+        it 'exists' do
+            #test several elements in the game objects array to ensure they're all instances of Games
+            expect(@game_objects[0]).to be_an_instance_of(Games)
+            expect(@game_objects[1]).to be_an_instance_of(Games)
+            expect(@game_objects[2]).to be_an_instance_of(Games)
+            expect(@game_objects[3]).to be_an_instance_of(Games)
+        end
+
+        it 'has attributes' do
+
+            expect(@game_objects[0].game_id).to eq("2012030221")
+            expect(@game_objects[0].home_team_id).to eq("6")
+            expect(@game_objects[1].away_goals).to eq(2)
+            expect(@game_objects[1].game_id).to eq("2012030222")
+        end
+    end
+
+    describe '#self.load_csv' do 
+        it 'creates games from csv' do
+            #when given a file pathway string, it will create objects using the data in the csv file'
+            game_path = './data/games.csv'
```

Comment:
make sure you are using the new fixture file you created here and not the original! Otherwise the purpose of the fixture files becomes null

Reply:
Corrected!